src/hardware/input/mouse_config.cpp

```diff
@@ -20,6 +20,10 @@ CHECK_NARROWING();
 MouseConfig mouse_config;
 MousePredefined mouse_predefined;
 
+constexpr auto BuiltinDosDriverOff   = "off";
+constexpr auto BuiltinDosDriverOn    = "on";
+constexpr auto BuiltinDosDriverNoTsr = "no-tsr";
```

Comment:
nitpicking: built-in, so "BuiltIn" 😆

Reply:
Consider namespacing these instead of prefixing.

Reply:
OK, changed.